src/pages/map/Map.tsx

```
@@ -0,0 +1,107 @@
+import { useEffect, useRef, useState } from "react";
+import ReactDOMServer from "react-dom/server";
+import { LocateIcon, StarIcon } from "lucide-react";
+import CustomMarker from "@/components/common/CustomMarker";
+import MapDetail from "./MapDetali";
+
+export default function Map() {
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)